plugin-maven/build.gradle

```diff
@@ -1,42 +1,15 @@
-buildscript {
-	repositories { mavenCentral() }
-	dependencies { classpath "com.github.spullara.mustache.java:compiler:${VER_MUSTACHE}" }
-}
 plugins {
-	id 'cz.malohlava.visteg' version '1.0.5' // https://github.com/mmalohlava/gradle-visteg
+	// https://www.benediktritter.de/maven-plugin-development/#release-history
+	id 'de.benediktritter.maven-plugin-development' version '0.4.0'
 }
+
 repositories { mavenCentral() }
 apply from: rootProject.file('gradle/changelog.gradle')
-apply from: rootProject.file('gradle/spotless-freshmark.gradle')
-
-// to generate taskGraph.pdf
-// - set enabled (below) to true
-// - run: ./gradlew :plugin-maven:test
-// - run: rm plugin-maven/output.pdf
-// - run: dot -Tpdf plugin-maven/build/reports/visteg.dot > plugin-maven/taskGraph.pdf
-visteg {
-	enabled = false
-	nodeShape = 'box'
-	startNodeShape = 'box'
-	endNodeShape = 'box'
-	colorscheme = 'pastel24' // https://www.graphviz.org/doc/info/colors.html
-}
-
-import com.github.mustachejava.DefaultMustacheFactory
-
-import java.nio.file.Files
-
-import static java.nio.charset.StandardCharsets.UTF_8
-import static java.nio.file.StandardOpenOption.CREATE
-import static java.nio.file.StandardOpenOption.TRUNCATE_EXISTING
-
 ext.artifactId = project.artifactIdMaven
 version = spotlessChangelog.versionNext
-apply from: rootProject.file("gradle/java-setup.gradle")
-apply from: rootProject.file("gradle/java-publish.gradle")
 
-final MAVEN_PROJECT_DIR = project.layout.buildDirectory.dir("mavenProject").get()
-final LOCAL_MAVEN_REPO_DIR = project.layout.buildDirectory.dir("localMavenRepository").get()
+apply from: rootProject.file("gradle/java-setup.gradle")
+apply from: rootProject.file('gradle/spotless-freshmark.gradle')
 
 def mvnw(String args) {
```

Comment:
Is it now possible to remove this method?

Reply:
Good call! Done.